podcasts/UpNextViewController.swift

```diff
@@ -132,12 +130,12 @@ class UpNextViewController: UIViewController, UIGestureRecognizerDelegate {
         remainingLabel.adjustsFontSizeToFitWidth = true
         remainingLabel.minimumScaleFactor = 0.8
         remainingLabel.numberOfLines = 2
-        remainingLabel.style = .playerContrast02
+        remainingLabel.style = .primaryText02
         remainingLabel.themeOverride = themeOverride
 
         clearQueueButton.setTitle(L10n.queueClearQueue, for: .normal)
-        clearQueueButton.setTitleColor(AppTheme.colorForStyle(.playerContrast01, themeOverride: themeOverride), for: .normal)
-        clearQueueButton.setTitleColor(AppTheme.colorForStyle(.playerContrast05, themeOverride: themeOverride), for: .disabled)
+        clearQueueButton.setTitleColor(AppTheme.colorForStyle(.primaryText02, themeOverride: themeOverride), for: .normal)
+        clearQueueButton.setTitleColor(AppTheme.colorForStyle(.primaryText02, themeOverride: themeOverride).withAlphaComponent(0.5), for: .disabled)
```

Comment:
I think the `primaryInteractive01` and `primaryInteractive01Disabled` colors are better here than the more neutral text color. They make it clearer that this is a button the user can interact with, and using these colors seems more consistent with the rest of the UI in the app.

This is what it looks like with the interactive colors on Android with the light theme.

| active | disabled |
| --- | --- |
| ![Screenshot 2023-12-08 at 4 42 35 PM](https://github.com/Automattic/pocket-casts-ios/assets/4656348/51aa9927-53e8-4fbc-87e9-44156264ce27) | ![Screenshot 2023-12-08 at 4 44 23 PM](https://github.com/Automattic/pocket-casts-ios/assets/4656348/b13c3e38-8a6d-4134-901f-88fdceaa1995) |

Reply:
Ok, I'll do the same on Android.